test/money_test.exs

```diff
@@ -339,6 +339,26 @@ defmodule MoneyTest do
     end
   end
 
+  test "to_string configuration with custom_display_options" do
```

Comment:
Please add more tests to make sure that `custom_display_options` is prioritized over `common display options`, and `opts` is prioritized over both of them

Reply:
done!